src/main/java/com/iota/iri/service/spentaddresses/impl/SpentAddressesProviderImpl.java

```diff
@@ -85,26 +76,16 @@ private void readPreviousEpochsSpentAddresses() {
                 readSpentAddressesFromStream(
                         SpentAddressesProviderImpl.class.getResourceAsStream(previousEpochsSpentAddressesFile));
             } catch (SpentAddressesException e) {
-                log.error("failed to read spent addresses from " + previousEpochsSpentAddressesFile, e);
+                log.error("Failed to read spent addresses from " + previousEpochsSpentAddressesFile, e);
```

Comment:
We might want to throw the error here, and not catch it in the constructor.  Pretty important if were missing/failing to read this file

Reply:
It was like this before. I didn't catch Hans on that...
Good job :-)